skbeam/core/accumulators/binned_statistic.py

```diff
@@ -317,11 +319,14 @@ def __call__(self, values, statistic=None):
                 np.seterr(**old)
             self.result.fill(null)
 
-            # Sort by bin number
-            idx = self.xy.argsort()
-            vfs = values[idx]
+            if not self.argsort_index:
```

Comment:
You can't convert a numpy array directly to a truth value:

`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`

Use `self.argsort_index is None` instead. Same for self.bincount below.

Reply:
Ah yes I had forgot about numpy truthiness